src/contracts/address-mapper.ts

```diff
@@ -54,6 +54,7 @@ export class AddressMapper extends Contract {
     return this.callAsync<void>('AddIdentityMapping', mappingIdentityRequest)
   }
 
+  //Similar to addIdentityMappingAsync but for Binance, and this function not require message encryption
```

Comment:
That's not true, it doesn't require message hashing since that's presumably done by the BinanceSigner. Also see the other method comments, they use `/** */` comments, use that.